### PR TITLE
exp/clients/horizon: support trade aggregations

### LINKED ISSUE #1060: exp/clients/horizon: trade aggregation
Support trade aggregation endpoint

### PR DESCRIPTION
**Pull request by poliha**
This PR add support for querying trade aggregations.
Closes #1060

### CODE REVIEW

File: exp/clients/horizon/main.go [hunk 1]
```diff
@@ -234,3 +235,19 @@ type PathsRequest struct {
 	DestinationAmount      string
 	SourceAccount          string
 }
+
+// TradeAggregationRequest struct contains data for getting trade aggregations from an horizon server
+type TradeAggregationRequest struct {
+	StartTime          string
+	EndTime            string
```
Comment (bartekn) [inline exp/clients/horizon/main.go]: I'm wondering if we should change it to `time.Time`. Then users will be able to create periods quickly like:
```go
endTime := time.Now()
TradeAggregationRequest{
    StartTime: endTime.Add(-time.Hour),
    EndTime: endTime,
}
```

File: exp/clients/horizon/main.go [hunk 2]
```diff
@@ -234,3 +235,19 @@ type PathsRequest struct {
...
+	Resolution         string
+	Offset             string
```
Comment (bartekn) [inline exp/clients/horizon/main.go]: ...and these could be `time.Duration`. Ex:
```go
TradeAggregationRequest{
    Resolution: 24 * time.Hour,
    Offset: time.Hour,
}
```

File: exp/clients/horizon/main.go (hunk 1 above)
Comment (poliha) [reply exp/clients/horizon/main.go]: Sounds good.. we can then convert this internally to unix time. 
Does horizon expect seconds or milliseconds?

Comment (bartekn) [reply exp/clients/horizon/main.go]: It's milliseconds.

File: exp/clients/horizon/main.go (hunk 2 above)
Comment (poliha) [reply exp/clients/horizon/main.go]: Actually we can even have a custom type for `Resolution` and have constants. There are only currently 5 values supported

Comment (bartekn) [reply exp/clients/horizon/main.go]: Good point! But still would be great if these were `time.Durations`.

**Pull request merged**